### ISSUE TITLE
A mistake

### ISSUE THREAD

**Issue by ReturnLab** [rifflock/lfshook#31]
Change for README.md
```
func NewLogger() *logrus.Logger {
	if Log != nil {
		return Log
	}

	pathMap := lfshook.PathMap{
		logrus.InfoLevel:  "/var/log/info.log",
		logrus.ErrorLevel: "/var/log/error.log",
	}

	// here need new a log . 
Log = logrus.New()
	Log.Hooks.Add(lfshook.NewHook(
		pathMap,
		&logrus.JSONFormatter{},
	))
	return Log
}
```

**Comment by rifflock** [comment]
You are correct. I updated your comment for formatting. I'll update the README for this and #30 this weekend.

**Comment by rifflock** [comment]
fixed in v2.3
